vehicles/src/common/enum/permit-search.enum.ts

```diff
@@ -1,7 +1,5 @@
 export enum PermitSearch {
   PLATE = 'plate',
   PERMIT_NUMBER = 'permitNumber',
-  CLIENT_NUMBER = 'clientNumber',
-  COMPANY_NAME = 'companyName',
   APPLICATION_NUMBER = 'applicationNumber',
```

Comment:
Searching by application number isn't an option either. Still needed?

https://github.com/bcgov/onRouteBCSpecification/blob/main/PPC%20Dashboard/Staff%20search%20for%20Permits.feature

Reply:
Out of syllabus :P

Will wait out for discovery before removing it